docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,610 +1,127 @@
-6. Select a name and location for you certificate and click through the wizard.
+## Step 2/2. Run the discovery wizard
 
-Now transfer the exported file to the system where you're running Teleport. You
-can either add this certificate to your system's trusted repository or provide
-the filepath to the `der_ca_file` configuration variable.
+In your web browser, access the teleport Web UI at <ScopedBlock scope={["oss", "enterprise"]}>`teleport.example.com`</ScopedBlock><ScopedBlock scope={["cloud"]}>`mytennant.teleport.sh`</ScopedBlock>. Click on your user name at the top right and select **Manage Access**, Select "Desktop" from the main menu, then **NEXT**:
 
-## Step 6/7. Configure Teleport
+If your Windows server is already running as an Active Directory controller, skip to the next step. Otherwise, copy and paste the first command provided into a Windows PowerShell window. If your server is not already running AD Certificate Services, copy and paste the second command after the first one completes and the server restarts:
 
-<Admonition type="note" title="Teleport CA">
-  Prior to v8.0, the Teleport CA was not compatible with Windows logins. If
-  you're setting up Desktop Access in an existing cluster created before v8.0,
-  you must first perform a [CA rotation](../management/operations/ca-rotation.mdx) in
-  order to resolve this.
-</Admonition>
+![Install Active Directory](../../img/desktop-access/install-active-directory.png)
 
-Install Teleport on the host where you will run the Teleport Desktop Service:
+Once the server is restarted from one or both command executions, click **NEXT**.
 
-(!docs/pages/includes/install-linux.mdx!)
+Copy and paste the provided command into a Windows PowerShell window to download and run the 'configure Active Directory' script:
 
-In order to enable Desktop Access in Teleport, add the following section in
-`/etc/teleport.yaml` on your Linux server. For a detailed description of these
-configuration fields, see the
-[configuration reference](./reference/configuration.mdx) page.
+![Configure Active Directory](../../img/desktop-access/configure-active-directory.png)
 
-<Tabs>
-<TabItem scope={["oss","enterprise"]} label="Self-Hosted">
-The Teleport Windows Desktop Service will establish a reverse tunnel to
-the Proxy Service. This requires setting `proxy_server` to your Proxy Service address and
-providing a join token.
+Click **NEXT**.
 
-First, generate a join token with the following command:
+The PowerShell script will output a Teleport configuration block. Copy this block to a temporary location. Click **Next**.
 
-```code
-$ tctl tokens add --type=windowsdesktop
-```
-
-Copy the join token to a file on the instance where you will run the Windows Desktop
-Service, and then use the following configuration:
+On the host you installed Teleport to run as the Desktop Access connector, edit `/etc/teleport.yaml` and paste the configuration provided by the output of the previous step:
 
 ```yaml
 version: v3
```

Comment:
Config version v3 is specific to Teleport 11+, so you'll need to do some manual updates here for the v10 backport.